Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1147,7 +1147,7 @@ open class NavigationMapView: UIView {
      Callouts text font also respects `NavigationMapView.routeDurationAnnotationFontNames` property.
      Default value is `true`.
      */
-    public var showsContinuousAlternativeRoutesRelativeDuration: Bool = true {
+    public var showsRelativeDurationOnAlternativeRoutes: Bool = true {
```

Comment:
Based on your explanation in https://github.com/mapbox/mapbox-navigation-ios/pull/3956#discussion_r907209097, I agree with reintroducing “continuous” into this property name, to keep developers from thinking it also applies to one-shot alternatives: `showsRelativeDurationOnContinuousAlternativeRoutes`.